building/build-debs/homeworld-apt-setup/build-package.sh

```diff
@@ -2,4 +2,8 @@
 set -eu
 source ../common/package-build-helpers.sh
 
+sed "s;\\\${APT_REPOSITORY};${APT_REPOSITORY};g" < homeworld.sources.in > homeworld.sources
```

Comment:
Might be worth doing this in a way that doesn't require that a specific character is not present in the value of ${APT_REPOSITORY} -- or at least will properly error out in that case.

Reply:
It would be nice if Bash had a cleaner way to do substitutions in files. Is this a security concern, or are we just handling the possibility of having special characters in future apt branches? I could use something from https://stackoverflow.com/a/2705678/1263980, but it might be easier (and better) to just limit the characters allowed in apt branch names, like to [A-Za-z0-9_-].

Reply:
Just limiting the characters sounds good to me.